src/main/java/com/manardenza/entity/Room.java

```diff
@@ -2,8 +2,10 @@
 
 import lombok.Data;
 
+import java.io.Serializable;
+
 @Data
-public class Room extends AbstractObject {
+public class Room extends AbstractObject implements Serializable{
```

Comment:
If you have marked AbstractObject as Serializable, this class will become so as well